src/pudl/metadata/sources.py

```diff
@@ -835,5 +835,41 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "vceregen": {
+        "title": "Vibrant Clean Energy Renewable Generation Profiles",
+        "path": "https://vibrantcleanenergy.com/products/datasets/",
```

Comment:
Was not sure which link to put here because the data are proprietary. Could have linked to either of these PDFs as well: 

https://vibrantcleanenergy.com/wp-content/uploads/2021/04/2018%20VCE%20Study_Dataset%20Methods%20and%20Analysis536960.pdf 

https://vibrantcleanenergy.com/wp-content/uploads/2020/08/VCE-Weather-Dataset-Overview_August2020.pdf

Reply:
I asked Chris in email which was the appropriate document and he said the [Dataset Overview](https://vibrantcleanenergy.com/wp-content/uploads/2020/08/VCE-Weather-Dataset-Overview_August2020.pdf) (Aug, 2020) was the better one to reference.  I also asked permission to archive it alongside the inputs (or the way we do in our GitHub repo) so that when the VCE website goes down it's still available.

Reply:
Will we be able to commit a large PDF to git, space-wise?

Reply:
Has he given us the go-ahead to archive the PDF or are we still waiting for a response?

Reply:
It's public on the website so I don't see why it would be a problem.